spring-xd-integrations.tex

```diff
@@ -1,68 +1,88 @@
-A counter/gauge is a specialized sink increments a field on a datastore every time a 
-message is received refer to the (Analytics Section) for more information.  A delegate 
-sink's responsibility is to translate the XD message to the format that can be used by the
- external agent and dispatch that data to the agent or store the data in the specified 
- repository. \par  
- The basic sink is comprised of a "input" channel and a integration bean.
- The "input channel " will receive all messages from the stream and will and dispatch the
- messages to the integration bean. figure~\ref{fig:sinkmbc}. It is the responsibility of 
- the integration bean to implement retry behavior in cases of failure. In the current set 
- of XD sinks this behavior can be configured when the user constructs the stream by setting
- the parameters for the sink.  Each sink has different features so please review the 
- documentation on what features are available.\par
+Sink modules convert and deliver messages from a stream in a format consumable by an external application.
+There are two types of sinks: counter/gauge and delegate.
+A counter/gauge is a specialized sink that increments a field on a datastore every time a
+message is received. (Refer to the Analytics section for more information.)  A delegate
+sink translates messages to the format expected by the external application.
+After transforming the message, the resulting data is sent to the external application.
+
+\par
+
+The basic sink is comprised of a "input" channel and an integration bean.
+The input channel receives all messages from the stream and dispatches
+messages to the integration bean (see figure~\ref{fig:sinkmbc}.) It is the responsibility of
+the integration bean to implement retry behavior in cases of failure. The sinks included
+in Spring XD have configurable options for retries.
+
+\par
+
 \begin{figure}
 \centering
 \epsfig{file=integration-module-input-channel.eps, height=.8in, width=2in}
 \caption{Sink Module Basic Components}
 \label{fig:sinkmbc}
 \end{figure}
+
 \subsection{Module Integrations: Job}
-Spring XD uses Spring batch \cite{spring-batch-reference} as its foundation for implementing
-job modules.  Jobs enables users to execute enterprise batch processes within the XD application. 
-Jobs are typically used when running long lasting enterprise tasks that are transactionally 
-sensitive.  Thus in cases that the job fails, a job can be designed to be restarted and 
-pickup where it left off or rollback the changes that were in transaction.\par
-A job is typical comprised of a job definition along with the supporting 
+Spring XD uses Spring Batch \cite{spring-batch-reference} as the foundation for implementing
+job modules.  Jobs enables users to execute enterprise batch processes within Spring XD.
+Jobs are typically used when running long lasting tasks that have transactional requirements.
+Thus in cases that the job fails, a job can be designed to be restarted and
```

Comment:
This seems to imply that jobs that can pick up where they left off don't use transactions - is this accurate? I had a bit of trouble understanding the part starting with "Thus in cases that the job fails..." so I left it alone for now.